In-process and VS Auto build on browser request (#9943)
aspnet/Docs: aspnetcore/migration/21-to-22.md

Change the `FROM` lines in your *Dockerfile* to use the new image tags in the preceding table's 2.2 column.

> ## Visual Studio Auto build on browser request with IIS in-process hosting

Reviewer: This heading should start with an action verb, as the other headings do.

Author: Any I-dears? This section is stating a fact of capabilities and not asking the reader to take any specific action. It's just saying "don't expect this to work." What's the action verb/verbal phrase that makes sense in this scenario? 🤔 ...

1. Avoid Visual Studio Auto build on browser request with IIS in-process hosting
1. Build manually in Visual Studio when using IIS in-process hosting
1. Observe regression with Visual Studio Auto build on browser request and IIS in-process hosting
1. Note regression with Visual Studio Auto build on browser request and IIS in-process hosting

🤔 ...... so far, I like number 2 ... shorter. Which one do you like, or should I keep brainstorming?

Change the `FROM` lines in your *Dockerfile* to use the new image tags in the preceding table's 2.2 column.

## Build manually in Visual Studio when using IIS in-process hosting

> Visual Studio's **Auto build on browser request** experience doesn't function with the [IIS in-process hosting model](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model). You must manually rebuild the project when using in-process hosting. We plan to improve this experience in a future release of Visual Studio.

Reviewer: ```suggestion
Visual Studio's **Auto build on browser request** experience doesn't function with the [IIS in-process hosting model](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model). You must manually rebuild the project when using in-process hosting. This experience will improve in a future release of Visual Studio.
```

Author: I wrote the sentence that way specifically to avoid future tense. I also prefer to say that engineering "plans" to address it.

Author: > We're looking to improve that in VS2019.

"looking to" == "plan to"

Reviewer: I was just trying to avoid "we".

Author: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we

If "we recommend" is ok, then I think "we plan" is ok, too.

Author: We can't make the community member the focus of this line. It's basically has to say that MS is doing something here imo.

Author: > A plan is in place to improve this experience in a future release of Visual Studio.

🎤*drop* :smile:

Author: *Ah! I got it!* ....

> Improvements to this experience are planned for a future release of Visual Studio.
